org.faktorips.aggregator/pom.xml

```diff
@@ -1,153 +1,129 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
-    xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <modelVersion>4.0.0</modelVersion>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
 
-    <properties>
-        <git.user>${user.name}</git.user>
-        <java.version>11</java.version>
-        <target-platform>eclipse-2019-03</target-platform>
-        <checkstyle.header.file>${project.basedir}/../org.faktorips.aggregator/checkstyle.java.header</checkstyle.header.file>
-    </properties>
+	<parent>
+		<groupId>org.faktorips</groupId>
+		<artifactId>base</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
 
-    <groupId>org.faktorips</groupId>
-    <artifactId>org.faktorips.aggregator</artifactId>
-    <version>22.6.0-SNAPSHOT</version>
-    <packaging>pom</packaging>
+	<properties>
+		<git.user>${user.name}</git.user>
+		<checkstyle.header.file>${project.basedir}/../org.faktorips.aggregator/checkstyle.java.header</checkstyle.header.file>
+	</properties>
 
-    <parent>
-        <groupId>de.faktorzehn</groupId>
-        <artifactId>faktorzehn-tycho-build</artifactId>
-        <version>1.2.8</version>
-    </parent>
+	<artifactId>org.faktorips.aggregator</artifactId>
+	<version>22.6.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
 
-    <!-- These are all modules that are included in the Faktor-IPS build -->
 
-    <modules>
-        <module>targets</module>
-        <module>features</module>
-        <module>plugins</module>
-        <module>nexus8</module>
-        <module>nexus11</module>
-        <module>tests</module>
-        <module>../faktorips-maven-archetype</module>
-        <module>../faktorips-maven-plugin</module>
-        <module>../faktorips-documentation</module>
-        <module>../faktorips-schemas</module>
-        <module>sites</module>
-        <module>../faktorips-coverage</module>
-    </modules>
+	<!-- These are all modules that are included in the Faktor-IPS build -->
 
-    <scm>
-        <connection>scm:git:ssh://${git.user}@projekte.faktorzehn.de/projekte/faktorips/faktorips.base.git</connection>
-    </scm>
+	<modules>
+		<module>targets</module>
+		<module>plugins</module>
+		<module>features</module>
+		<module>tests</module>
+		<!-- <module>nexus8</module> -->
+		<!-- <module>nexus11</module> -->
+<!-- 		<module>../faktorips-maven-archetype</module> -->
```

Comment:
Das ist jetzt erst mal raus, weil work-in-progress?

Reply:
Ja da fhelt wie geschrieben derzeit z.B. noch das config-project, das füge ich jetzt ermal als erstes hinzu.